### PR TITLE
Add case to not skip gatekeeper tests

### PR DESCRIPTION
**Pull request by JustinKuli**
The sample gatekeeper policy created during the tests makes an admission
webhook that prevents the server-side dry-run from testing whether an
openshift-* namespace can be created. But if gatekeeper (and this rule)
are already present, then we can assume we would be able to make the ns.

Refs:
 - https://github.com/open-cluster-management/backlog/issues/13759

Signed-off-by: Justin Kulikauskas <jkulikau@redhat.com>

### CODE REVIEW

**Comment by JustinKuli** [comment]
/hold

I'll wait for the e2e tests to run this time before merging

**Review by dhaiducek — APPROVED** [review]

**Comment by JustinKuli** [comment]
/unhold

nevermind, the PR tests don't look like they run these anyway

**Pull request merged**
